File: LLM-Translations/GPT-4/codenet/Java/Go/s739337307.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var n, m int
	fmt.Scan(&n)
	fmt.Scan(&m)

	mapQuestion := make(map[int]int)

	countAc := 0
	countWaAll := 0

	for i := 0; i < m; i++ {
		var question int
		var waOrAc string
		fmt.Scan(&question)
		fmt.Scan(&waOrAc)

		countWa, exists := mapQuestion[question]

		if !exists {
			if waOrAc == "WA" {
				mapQuestion[question] = 1
			} else {
				mapQuestion[question] = -1
				countAc++
			}
		} else {
			if countWa < 0 {
				continue
			} else {
				if waOrAc == "WA" {
					mapQuestion[question] = countWa + 1
				} else {
					mapQuestion[question] = -countWa
					countAc++
					countWaAll += countWa
				}
			}
		}
	}

	fmt.Println(strconv.Itoa(countAc) + " " + strconv.Itoa(countWaAll))

}
// 

File: LLM-Translations/GPT-4/codenet/Java/C/s739337307.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    int n, m;
    scanf("%d %d", &n, &m);

    int map[100001] = {0};

    int countAc = 0;
    int countWaAll = 0;

    for (int i = 0; i < m; i++) {
        int question;
        char waOrAc[3];
        scanf("%d %s", &question, waOrAc);

        if (map[question] == 0) {
            if (strcmp(waOrAc, "WA") == 0) map[question] = 1;
            else {
                map[question] = -1;
                countAc++;
            }
        } else {
            int countWa = map[question];
            if (countWa < 0);
            else {
                if (strcmp(waOrAc, "WA") == 0) map[question] = countWa + 1;
                else {
                    map[question] = -countWa;
                    countAc++;
                    countWaAll += countWa;
                }
            }
        }
    }

    printf("%d %d\n", countAc, countWaAll);

    return 0;
}
// 
